src/index.js

```diff
@@ -67,7 +67,7 @@ function nowApi () {
         if (r.state === 'READY') {
           return r
         }
-        if (r.state === 'DEPLOYING' || r.state === 'BOOTED') {
+        if (~['DEPLOYING', 'BOOTED', 'BUILDING'].indexOf(r.state)) {
```

Comment:
I used to hate it as well until I realized that `~.indexOf()` could be read as a  "contains" and forget about the fact that it returns an integer... . and now it's a habit since I mostly use `.indexOf` like `.contains` anyway.